kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigReader.java

```diff
@@ -413,10 +426,6 @@ else if (args[0].equals("iface") && args[1].equals(interfaceName)) {
 
         } catch (FileNotFoundException err) {
             throw new KuraException(KuraErrorCode.INTERNAL_ERROR, err);
-        } finally {
```

Comment:
`scanner` is declared as a resource in a try-with-resources that automatically closes it at the end of the statement.